BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BatchManagement",
+    "version": "2017-05-01",
+    "x-ms-code-generation-settings": {
+      "name": "BatchManagementClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}": {
+      "put": {
+        "tags": [
+          "BatchAccount"
+        ],
+        "operationId": "BatchAccount_Create",
```

Comment:
As we use the `NOUN` part of `NOUN_VERB` pattern in `operationId` in the grouping of operation. Currently, we have a model named as `NOUN` that'll be auto resolved by autorest to resolve naming conflicts. 

Reference: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/254658613#L685

Please make sure that is what you expect in SDK or we need to make them differ here.